### PR TITLE
CLOUDP-222191: Update quick start guide and provided samples

### PR DESCRIPTION
**Pull request by helderjs**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

I replaced the hard-coded examples with references from inside our config sample folder. Ensuring both work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,19 @@ The full documentation for the Operator can be found [here](https://docs.atlas.m
...
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+2. Have a Kubernetes solution available to use.
+   If you need a Kubernetes solution, see the [Kubernetes documentation on picking the right solution](https://kubernetes.io/docs/setup). For testing, MongoDB recommends [Kind](https://kind.sigs.k8s.io/).
+3. Choose the version of the operator you want to run
+
 ### Step 1. Deploy Kubernetes operator using all in one config file
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/mongodb/mongodb-atlas-kubernetes/main/deploy/all-in-one.yaml
+kubectl apply -f https://raw.githubusercontent.com/mongodb/mongodb-atlas-kubernetes/<operator_version>/deploy/all-in-one.yaml
```
Comment (s-urbaniak) [inline README.md:26]: That's clever :-)

File: README.md [hunk 2]
```diff
@@ -11,10 +11,19 @@ The full documentation for the Operator can be found [here](https://docs.atlas.m
 
 ## Quick Start guide
 
+### Prerequisites to Install using kubectl
+
+Before you install the MongoDB Atlas Operator using `kubectl`, you must:
+
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+2. Have a Kubernetes solution available to use.
```
Comment (s-urbaniak) [inline README.md:19]: I'd say just:
```
2. Have Kubernetes installed, i.e. [Kind](https://kind.sigs.k8s.io/)
```

File: README.md [hunk 3]
```diff
@@ -11,10 +11,19 @@ The full documentation for the Operator can be found [here](https://docs.atlas.m
...
+   If you need a Kubernetes solution, see the [Kubernetes documentation on picking the right solution](https://kubernetes.io/docs/setup). For testing, MongoDB recommends [Kind](https://kind.sigs.k8s.io/).
```
Comment (s-urbaniak) [inline README.md:20]: let's remove recommendations

File: README.md [hunk 4]
```diff
@@ -11,10 +11,19 @@ The full documentation for the Operator can be found [here](https://docs.atlas.m
...
+3. Choose the version of the operator you want to run
```
Comment (s-urbaniak) [inline README.md:21]: How can we find out which versions there are?

**Comment by s-urbaniak** [comment]
What do we do about the quickstart guide in the CSV? https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml#L70-L176

**Comment by s-urbaniak** [comment]
@helderjs that's a great start, thank you! I have a couple of comments and a question regarding the CSV.

**Comment by helderjs** [comment]
> What do we do about the quickstart guide in the CSV? https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml#L70-L176

Thanks for reminding me about it. Looks like it was never updated, the information there was from before GA :grimacing:

File: README.md [hunk 5]
```diff
@@ -11,10 +11,18 @@ The full documentation for the Operator can be found [here](https://docs.atlas.m
 
 ## Quick Start guide
 
+### Prerequisites to Install using kubectl
+
+Before you install the MongoDB Atlas Operator using `kubectl`, you must:
+
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+2. Have Kubernetes installed, i.e. [Kind](https://kind.sigs.k8s.io/)
```
Comment (josvazg) [inline README.md:19]: Or have access to a kubernetes cluster configuration right?

**Review by josvazg — APPROVED** [review]
LGTM, I assume we have followed the new steps successfully right?

File: config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml [hunk 6]
```diff
@@ -60,101 +60,96 @@ spec:
...
   description: |
-    The MongoDB Atlas Operator provides a native integration between the Kubernetes orchestration platform and MongoDB Atlas —
-    the only multi-cloud document database service that gives you the versatility you need to build sophisticated and resilient applications that can adapt to changing customer demands and market trends.
-
-    > Current Status: *Stable*. The Operator gives users the ability to provision
-    > Atlas projects, clusters and database users using Kubernetes Specifications and bind connection information
-    > into applications deployed to Kubernetes.
+    The MongoDB Atlas Operator provides a native integration between the Kubernetes orchestration platform and MongoDB Atlas
+    — the only multi-cloud document database service that gives you the versatility you need to build sophisticated and
+    resilient applications that can adapt to changing customer demands and market trends.
```
Comment (s-urbaniak) [inline config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml:65]: @helderjs please take this text in a consumable form and let it ratify by @dan-mckean

Comment (helderjs) [reply config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml:65]: @s-urbaniak this is a copy & paste from README, exactly the same content.

**Comment by s-urbaniak** [comment]
The CSV contains copy&paste from the README. We need a ratified version by @dan-mckean here instead.

File: README.md [hunk 7]
```diff
@@ -9,147 +9,25 @@ resilient applications that can adapt to changing customer demands and market tr
...
-## Quick Start guide
+## Getting Started
 
-### Step 1. Deploy Kubernetes operator using all in one config file
+### Supported features
 
-```
-kubectl apply -f https://raw.githubusercontent.com/mongodb/mongodb-atlas-kubernetes/main/deploy/all-in-one.yaml
-```
+* Create and configure a project, or connect to an existing one.
```
Comment (s-urbaniak) [inline README.md:16]:
```suggestion
* Create and configure an Atlas Project, or connect to an existing one.
```

File: README.md [hunk 8]
```diff
@@ -9,147 +9,25 @@ resilient applications that can adapt to changing customer demands and market tr
...
+* Deploy, manage, scale, and tear down clusters.
```
Comment (s-urbaniak) [inline README.md:17]:
```suggestion
* Deploy, manage, scale, and tear down Atlas clusters.
```

File: README.md [hunk 9]
```diff
@@ -9,147 +9,25 @@ resilient applications that can adapt to changing customer demands and market tr
...
+* Support for serverless instances.
```
Comment (s-urbaniak) [inline README.md:18]:
```suggestion
* Support for Atlas serverless instances.
```

File: README.md [hunk 10]
```diff
@@ -9,147 +9,25 @@ resilient applications that can adapt to changing customer demands and market tr
...
+* Manage IP Access Lists, network peering and private endpoints.
+* Configure and control Atlas’s fully managed cloud backup.
+* Configure federated authentication for your Atlas organization
+* Integrate Atlas monitoring with Prometheus.
 
-### Step 2. Create Atlas Deployment
+... and more.
 
-**1.** Create an Atlas API Key Secret
+To view the list of custom resources and each CR's schema, visit our [reference](https://www.mongodb.com/docs/atlas/operator/stable/custom-resources/)
```
Comment (s-urbaniak) [inline README.md:27]:
```suggestion
To view the list of custom resources and their respective schemas, visit our [reference](https://www.mongodb.com/docs/atlas/operator/stable/custom-resources/) 
```

**Review by s-urbaniak — APPROVED** [review]

**Pull request merged**